force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -1,368 +1,466 @@
+    return numsList; // Replace null with the variable you used to store the result
+  }
+
+  /**
+   * Question 14
+   * Remove duplicate values from the list of Strings.
+   * The returned Set contains the unique strings from the input List.
+   * Example: createStringSet(['a', 'b', 'a', 'c']) should return ['a', 'b', 'c']
+   * @param inputList The List of strings.
+   * @return A Set of unique strings.
+   */
+  public static Set<String> createStringSet(List<String> inputList) {
+    // Create a new Set of strings
+    Set<String> stringSet = new Set<String>();
+
+    //add strings to the set
+    stringSet.addAll(inputList);
+
+    return stringSet; // Replace null with the variable you used to store the result
+  }
+
+  /**
+   * Question 15
+   * Iterates over a list of integers and sums only the positive integers.
+   * Example: sumPositiveIntegers() should return 12
+   * @return The sum of positive integers from the list.
+   */
+  public static Integer sumPositiveIntegers() {
+    List<Integer> numbers = new List<Integer>{ -1, 2, -3, 4, -5, 6 }; //DO NOT CHANGE
+
+    // Loop through the list of integers
+    // if the number is negative skip this iteration
+    Integer sum = 0;
+
+    for (Integer i = 0; i < numbers.size(); i++) {
+      if (numbers[i] < 0) {
+        continue;
+      } else {
+        sum += numbers[i];
+      }
     }
 
-    /**
-     * Question 17
-     * Loop through the list of money and adds them up until it accumulates more than 40 and then stops counting.
-     * The change values are in cents and the total is to be calculated in dollars.
-     * Example: countMoney() should return 48.02
-     * @return The total value in dollars, stopped at the point when it exceeds 40 dollars.
-     */
-    public static Decimal countMoney() {
-        // The list of money in the wallet, represented in dollars and cents
-        List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
-
-        return null; // Replace null with the variable you used to store the result
+    return sum; // Replace null with the variable you used to store the result
+  }
+
+  /**
+   * Question 16
+   * Iterates over a list of strings and searches for a specific word. Once found, it breaks the loop.
+   *  Example: findWordInList("World", ['Hello', 'World', 'Goodbye']) should return 1
+   * @param wordToFind The word to find in the list.
+   * @param words The list of words to search.
+   * @return The position of the word in the list as a string or -1 if the word was not found.
+   */
+  public static Integer findWordInList(String wordToFind, List<String> words) {
+    // The variable to store the index of the word
+    Integer wordPosition = -1;
+
+    // Loop through the list of words
+    for (Integer i = 0; i < words.size(); i++) {
+      // If the current word is the word to find, exit the loop
+      if (words[i].equals(wordToFind)) {
```

Comment:
Bracket notation is a bit of an oddity in Apex. It does work, but you will more commonly see people using .get(Index). I know what is more characters, so it would suggest is it more desirable like this, but it is sort of a quirk of best practice.